### DISCUSSION TITLE
No output

### DISCUSSION THREAD

**Discussion by danielchang1985** [explosion/spacy-llm#408] [Q&A]
Hi I tried setting up the environment and running the instructions as follows but there is not output from the command.

python run_pipeline.py "Matthew and Maria went to Japan to visit the Nintendo headquarters" ./zeroshot.cfg
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Text: Matthew and Maria went to Japan to visit the Nintendo headquarters
Entities: []

**Comment by rmitsch** [comment]
Hi @danielchang1985, have you inspected the prompt output? Which model are you using? It's possible that the model is failing to spot entities or produces a non-parseable output.
